tachyon/math/finite_fields/generator/prime_field_generator/prime_field_x86.h.tpl

```diff
@@ -218,19 +217,21 @@ class PrimeField<_Config, std::enable_if_t<_Config::%{asm_flag}>> final
   // MultiplicativeGroup methods
   constexpr std::optional<PrimeField> Inverse() const {
     PrimeField ret;
-    if (UNLIKELY(InvalidOperation(!value_.template MontgomeryInverse<Config::kModulusHasSpareBit>(
-            Config::kModulus, Config::kMontgomeryR2, ret.value_), "Inverse of zero attempted"))) {
-      return std::nullopt;
+    if (LIKELY(value_.template MontgomeryInverse<Config::kModulusHasSpareBit>(
+            Config::kModulus, Config::kMontgomeryR2, ret.value_))) {
```

Comment:
Is there a rule for putting/not-putting parentheses for single if-else code block? Maybe if it fits in a line then no parentheses and if not then put parentheses? Cuz above, I mostly saw without parentheses but ones that fit in a line.